src/domain/events/searchPanel/__tests__/SearchPanel.test.tsx

```diff
@@ -55,11 +65,21 @@ test('should search events with correct search params', async () => {
   const eventTypeCheckbox = screen.getByLabelText(/tapahtuma/i);
   await user.click(eventTypeCheckbox);
 
+  // Publisher filtering
```

Comment:
SonarCloud is detecting duplication here. Considering this whole test case, could there be a single common test utility function that does the test for both `src/domain/registrations/searchPanel/__tests__/SearchPanel.test.tsx` and `src/domain/events/searchPanel/__tests__/SearchPanel.test.tsx`?

Reply:
I added a separate shouldFilterEventsOrRegistrations function to test common search filters for events and registrations